Lab3.ipynb

```diff
@@ -0,0 +1,405 @@
+      "text/plain": [
+       "<PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=283x45 at 0x7F40B776E790>"
+      ]
+     },
+     "execution_count": 118,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 2.2: \n",
+    "print('5 sigma probability:', str(1-stats.norm.cdf(5)))\n",
+    "Image.open('Lab3.2.jpg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 2.2: Above is the probability of 5 sigma for a normal gaussian distribution, which translates to the integral of the pdf from a value x out to infinity. Since we want to know the distance corresponding to 5 sigma, we should look for the bounding value x. Since we are looking radially in 2-d space this distance isn't too difficult to calculate. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 119,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "d = 5.489\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"d =\", 5.489)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 3.1: What is the probability that the background data produces 6800 or less comsmic rays in 15 days?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 3.2: We need to fin dthe background pdf for a single minute then multiply that by 7200 minutes of observing the moon to get the proper background pdf. After that in order to get the probability we would observe 6800 or less counts of cosmic rays we would need to integrate the proper background pdf from negative infinity to 6800. After that we can calculate the sigma value corresponding to that for a standard normal distribution. There is a unique aspect in this scenario because we technically don't need to integrate from negative infinity to 6800. We could integrate from 0 sighting to 6800, since it's impossible to have negative sightings."
```

Comment:
Problem 3.2:
- typo: "fin dthe" --> find the
- first sentence makes sense so far.
- math integration of a Poisson from 0 to 6800 makes sense, though you might not have to include the image of the integral. I think you should describe the expected background pdf being a Poisson in a bit more detail.
- I'm not sure I agree with the last two sentences, though I do agree with the negative sightings comment.